src/weights.jl

```diff
@@ -194,16 +194,20 @@ pweights(vs::RealArray) = ProbabilityWeights(vec(vs))
 end
 
 """
-    eweights(t::AbstractVector{<:Integer}, λ::Real)
-    eweights(t::AbstractVector{T}, r::StepRange{T}, λ::Real) where T
-    eweights(n::Integer, λ::Real)
+    eweights(t::AbstractVector{<:Integer}, λ::Real, [n::Integer]; scaled=false)
```

Comment:
Why do we need to allow passing `n` separately here? Isn't it always equal to `length(t)`?

Reply:
That doesn't always hold for cases where you call `eweights(t, r, λ)` below. In that case, you want `n` to be with respect the range `r` and not the elements `t` within `r`. The most common use case for us is passing a set of actual dates `t` and specifying our expect date range `r`. The previous code wouldn't handle `missing` entries in `t` correctly.